tasks/copy_to_glacier/README.md

```diff
@@ -178,15 +154,15 @@ The `copy_to_glacier` lambda function expects that the input payload has a `gran
 ```
 From the json file, the `filepath` shows the current S3 location of files that need to be copied over to glacier ORCA S3 bucket such as `"filename": "s3://orca-sandbox-protected/MOD09GQ/006/MOD09GQ.A2017025.h21v00.006.2017034065109.hdf"`.
 **Note:** We suggest that the `copy_to_glacier` task be placed any time after the `MoveGranulesStep`. It will propagate the input `granules` object as output, so it can be used as the last task in the workflow.
-See `schemas/input.json` for more information.
+See the schema [input file](https://github.com/nasa/cumulus-orca/blob/develop/tasks/copy_to_glacier/schemas/input.json) for more information.
```

Comment:
I would point to the master branch even though it is not there yet so they would only see the current release and not what may not be released yet.

Reply:
changed